src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -581,6 +633,12 @@ def build(self, notification_uuid: str | None = None) -> Union[SlackBlock, Slack
                 self.get_context_block(suggested_assignee_text[:-2])
             )  # get rid of comma at the end
 
+        # add suspect commit info
+        if self.event:
+            suspect_commit_text = get_suspect_commit_block(project, self.event)
```

Comment:
could do self.group.get_latest_event() when self.event doesn't exist since we already do that in some places